scripts/etc/udhcpc/default.script

```diff
@@ -19,7 +19,7 @@ PEERDNS_IF=eth0
 case "$1" in
   deconfig)
     # bring interface up, but with no IP configured:
-    ip addr flush dev $interface
+    ip -f inet addr flush dev $interface
```

Comment:
I personally prefer the `-4` flag (which is a shortcut for `-family inet`... :thought_balloon:  it's more intuitive (for me, at least)